joomla/joomla-cms
# [3.9] Backport jQuery Security Fix

### Summary of Changes
This PR backports a security-relevant fix in JQuery, see https://github.com/jquery/jquery/pull/4333

As the original issue and the fix are public anyway, JSST decided to do a public PR to allow more testers to look at the patch.


### Testing Instructions
1. Apply patch, open browser console, browser Joomla front- and backend and make sure no errors are shown with debug mode enabled and disabled
